README.md

```diff
@@ -824,7 +824,10 @@ All statements above will return false if used with `===`
 
 <a href="https://www.owasp.org/index.php/Top_10-2017_A2-Broken_Authentication" target="_blank"><img src="https://img.shields.io/badge/%E2%9C%94%20OWASP%20Threats%20-%20A9:Broken%20Authentication%20-green.svg" alt=""/></a>
 
-**TL;DR:** A brute force protection middleware such as [express-brute](https://www.npmjs.com/package/express-brute) should be used inside an express application to prevent brute force/dictionary attacks on sensitive routes such as /admin or /login based on request properties such as the username, or other identifiers such as body parameters
+**TL;DR:** A simple and powerful technique is to limit authorization attempts using two metrics:
+           
+1. The first is number of consecutive failed attempts by the same user unique ID/name and IP address.
+1. The second is number of failed attempts from an IP address over some long period of time. For example, block an IP address if it makes 100 failed attempts in one day.
```

Comment:
Minor - this should be 2.

Reply:
This is markdown syntax - if you preview this file, it looks fine.

Reply:
Yes, but it is better for the source to match the output imo

Reply:
ah, right. fixed.